build/media-suite_compile.sh

```diff
@@ -330,11 +330,16 @@ if [[ $curl = y ]]; then
     enabled mbedtls && curl=mbedtls
     [[ $curl = y ]] && curl=schannel
 fi
-_check=(libgnutls.{,l}a gnutls.pc)
-if enabled_any gnutls librtmp || [[ $rtmpdump = y ]] || [[ $curl = gnutls ]] &&
-    ! files_exist "${_check[@]}" &&
-    do_wget -h b1f3ca67673b05b746a961acf2243eaae0ffe658b6a6494265c648e7c7812293 \
-    "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-3.6.10.tar.xz"; then
+
+if [[ $ffmpeg != no ]] && enabled_any gnutls librtmp || [[ $rtmpdump = y ]] || [[ $curl = gnutls ]]; then
+   [[ -z "$gnutls_ver" ]] &&
+        gnutls_ver="$(clean_html_index "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/")" &&
+        gnutls_ver="$(get_last_version "$gnutls_ver" "xz$" '3\.6\.\d+(\.\d+)?')"
+    gnutls_ver="${gnutls_ver:-3.6.10}"
+   _check=(libgnutls.{,l}a gnutls.pc) 
+    if do_pkgConfig "gnutls = $gnutls_ver"; then    
+        do_wget -h b1f3ca67673b05b746a961acf2243eaae0ffe658b6a6494265c648e7c7812293 \
+        "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-3.6.10.tar.xz"
```

Comment:
This may become a problem when a new gnutls version is released and doesn't match 3.6.10

Reply:
Yes, indeed.
I'd consider to drop the hash code check and make it like this:
```
    if do_pkgConfig "gnutls = $gnutls_ver"; then    
        do_wget  "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-${gnutls_ver}.tar.xz"

Maybe replace the hash check by the .sig signatures available in the same dir https://www.gnupg.org/ftp/gcrypt/gnutls /v3.6/.